testutils/utils.go

```diff
@@ -120,6 +120,54 @@ func NewSecretUnStruct(opts ...Option) *unstructured.Unstructured {
 	return obj
 }
 
+func NewNodeUnStruct(opts ...Option) *unstructured.Unstructured {
+	obj := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"kind":       "Node",
+			"apiVersion": "v1",
+			"metadata": map[string]interface{}{
+				"name": "test1",
+			},
+		},
+	}
+
+	for _, opt := range opts {
+		if err := opt(obj); err != nil {
+			log.Fatal(err)
+		}
+	}
+	return obj
+}
+
+func NewPodUnStruct(opts ...Option) *unstructured.Unstructured {
+	obj := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"kind":       "Pod",
+			"apiVersion": "v1",
+			"metadata": map[string]interface{}{
+				"name":      "test1",
+				"namespace": "test1",
```

Comment:
Both can be provided as arguments to the function.

Reply:
yes, but I followed it to keep it consistent with rest of the file. There are `options` to rename the name and namespace.